### PR TITLE
added issue 76

### PR DESCRIPTION
**Pull request by vasylyeva12**
#76

### CODE REVIEW

**Review by rasnaut — DISMISSED** [review]

File: js_frontend/public/index.html [hunk 1]
```diff
@@ -15,6 +15,7 @@
             <button onclick="appendNumber(2)">2</button>
             <button onclick="appendNumber(3)">3</button>
             <button onclick="setOperation('+')">+</button>
+            <button onclick="setOperation('tan')">tang</button>
```
Comment (Parasolkasweet) [inline js_frontend/public/index.html]: Not correct 'tan", write 'tang'.

**Review by Parasolkasweet — CHANGES_REQUESTED** [review]
Please correct according to comments.

**Review by Parasolkasweet — APPROVED** [review]

**Review by rasnaut — APPROVED** [review]

File: js_frontend/public/index.html (hunk 1 above)
Comment (Parasolkasweet) [reply js_frontend/public/index.html]: fixed

**Comment by vasylyeva12** [comment]
fixed

**Pull request merged**